python/tvm/topi/x86/utils.py

```diff
@@ -18,9 +18,95 @@
 import tvm
 
 
-def get_fp32_len():
+def target_has_sse42(target):
```

Comment:
Just curious about why it's named `sse42` ... Is it for `sse4 & avx2` like the pr title, or minor version like `sse 4.2`?

And an unimportant suggestion which you can ignore is to merge all of these functions below to something like `target_has_attr(target, attr)` and list the candidates of attr in the doc string.

Reply:
sse4.2 is a latest standard of SSE type of instructions and supported in more processors than sse4/sse3 or sse2. It continues to be used in the latest Intel edge devices for IoT or low end segments (Atom based). If we come to the requirement distinguish more precisely, we probably will have to redesign this part.

as for suggestion to introduce the only function instead several ones - I consider this as valuable comment and in the the future if we want to check more features this should done. Not sure that it make sense to do in this PR